app/components/modals/event-revision.js

```diff
@@ -0,0 +1,10 @@
+import ModalBase from 'open-event-frontend/components/modals/modal-base';
+
+export default ModalBase.extend({
+  isSmall : true,
+  actions : {
+    toggleView() {
```

Comment:
what's this action for?

Reply:
my bad, it was left in place while I was testing something.
Fixed now.